src/Sulu/Bundle/ContentBundle/Twig/ContentTwigExtension.php

```diff
@@ -138,7 +138,10 @@ public function needsAddButtonFunction(PropertyInterface $property)
      */
     public function isMultipleTest($property)
     {
-        return $property->getMinOccurs() > 1;
+        return
+            !is_null($property->getMaxOccurs()) ||
+            $property->getMinOccurs() > 1 ||
+            $property->getMinOccurs() < $property->getMaxOccurs();
```

Comment:
the problem is with null values, like these cases:
```
<property name="name" type="text_line" maxOccurs="20">
<property name="name" type="text_line" minOccurs="2">
```

Ideally the condition should return true on these cases:
minOccurs = 0 maxOccurs = 1
minOccurs = 0 maxOccurs = 10
minOccurs = 1 maxOccurs = 2
minOccurs = 2 maxOccurs = null
minOccurs = null (that translate to 1) maxOccurs = 3

and return false on these cases:
minOccurs = 1 maxOccurs = 1
minOccurs = 0 maxOccurs = 0
minOccurs = 1 maxOccurs = null
minOccurs = 10 maxOccurs = 1

am I missing something?